sdf-feature.cddl

```diff
@@ -37,26 +37,26 @@ commonqualities = (
  ? sdfRequired: pointer-list    ; applies to qualities of properties, of data
 )
 
-; for building hierarchy
+; for single or nested things, or for arrays of things (1.2)
```

Comment:
One aspect of this PR I've been thinking about lately is the addition of the `minItems`/`maxItems` members to `sdfThing`. Allowing them also on `sdfThing`s could increase the potential complexity of models quite a lot as you could define `sdfThing` arrays within `sdfThing` arrays (and so on) and I am not sure if this is desirable.

Therefore, I've been wondering if it should be considered to keep `sdfObject` but allowing Interaction Affordances as well as `sdfData` in `sdfThing` (as I originally proposed in #56). This way, you could keep `sdfObject` as a definition for "leaf nodes" within a model as well as these kinds of array definitions while still increasing the expressiveness of SDF (which, in my opinion, is needed to map to linked/nested WoT TMs/TDs).

Reply:
I discussed this comment with @cabo two weeks ago and the conclusion was that nested arrays of `sdfThing`s are actually a feature. I'll therefore mark this conversation as resolved.